docs/website/docs/guides/deployment-configurations/bare-metal.md

```diff
@@ -15,7 +15,7 @@ Bare-metal deployment typically uses IREE's LLVM compiler target much like the
 [CPU configuration](./cpu.md), but using a limited subset of IREE's CPU HAL
```

Comment:
For L14 (for whatever reason I cannot select it. :( ), ".. uses IREE's LLVM compiler target backends ..."?